modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -167,12 +206,18 @@ INSFVMomentumAdvection::computeResidualsAndAData(const FaceInfo & fi)
     }
```

Comment:
in the previous branch, we could use this too. Ramiro does not need porouus media rn afaik

Reply:
actually the previous branch does not use the limiters for the porosity jump case